sdk/servicebus/service-bus/package.json

```diff
@@ -62,7 +62,7 @@
     "test:node": "npm run clean && npm run build:test:node && npm run integration-test:node",
     "test": "npm run test:node && npm run test:browser",
     "unit-test:browser": "echo skipped",
-    "unit-test:node": "npm run build:test:node && nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 120000 --exit --full-trace dist-esm/test/internal/**/*.spec.js",
+    "unit-test:node": "npm run build:test:node && nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 120000 --exit --full-trace dist-esm/test/internal/**/*.spec.js dist-esm/test/node/packageInfo.spec.js",
```

Comment:
Isnt there a sample.spec.js file also under "node"?

Reply:
Sure, that can be added too. 
Now, changed it to `node/*.spec.js` for unit-test command since these are the only two files, this would need an update if we ever add node-only live tests under "node/" folder.